src/Mod/Draft/DraftGeomUtils.py

```diff
@@ -1225,48 +1246,170 @@ def offsetWire(wire,dvec,bind=False,occ=False,widthList=None):
+                else:
+                    #just hardcoded default value as ArchWall would provide if dvec is not provided either
+                    delta = DraftVecUtils.scaleTo(delta, 200)
             except:
-                if widthList[0] > 0:
-                    delta = DraftVecUtils.scaleTo(dvec, widthList[0])	 	# to follow widthList[0]
-                delta = dvec
+                if dvec:
+                    delta = DraftVecUtils.scaleTo(delta, dvec.Length)
+                else:
+                    #just hardcoded default value as ArchWall would provide if dvec is not provided either
+                    delta = DraftVecUtils.scaleTo(delta, 200)
         else:
-              delta = dvec
+            delta = DraftVecUtils.scaleTo(delta,dvec.Length)
 
-        if i != 0:
-            if isinstance(curredge.Curve,Part.Circle):
-                v = curredge.tangentAt(curredge.FirstParameter)
-            else:
-                v = vec(curredge)
+        # Consider individual edge Align direction - ArchWall should now always provide alignList
 
-            ## TODO - 2019.6.16 - 'calculate' 'offset' direction (in vector) edge by edge instead of rotating previous vector based on dvec in future
-
-            angle = DraftVecUtils.angle(firstVec,v,norm)			# use vec deduced depending on geometry instead of - angle = DraftVecUtils.angle(vec(edges[0]),v,norm)
-            delta = DraftVecUtils.rotate(delta,angle,norm)
+        if i == 0:
+                if alignListC[0] == 'Center':
+                    delta = DraftVecUtils.scaleTo(delta, delta.Length/2)
+                #No need to do anything for 'Left' and 'Rigtht' as orginal dvec have set both the direction and amount of offset correct
+                #elif alignListC[i] == 'Left':  #elif alignListC[i] == 'Right':
+        if i != 0:
+            try:
+                if alignListC[i] == 'Left':
+                    curDir = 1
+                    curAlign = 'Left'
+                elif alignListC[i] == 'Right':
+                    curDir = -1
+                    curAlign = 'Right'
+                    delta = delta.negative()
+                elif alignListC[i] == 'Center':
+                    curDir = 1
+                    curAlign = 'Center'
+                    delta = DraftVecUtils.scaleTo(delta, delta.Length/2)
+            except:
+                curDir = firstDir
+                curAlign = firstAlign
+                if firstAlign == 'Right':
+                    delta = delta.negative()
+                elif firstAlign == 'Center':
+                    delta = DraftVecUtils.scaleTo(delta, delta.Length/2)
+
+        # Consider whether generating the 'offset wire' or the 'base wire'
+
+        if offsetMode == None:
+                # Consider if curOrientation and/or curDir match their firstOrientation/firstDir - to determine whether and how to offset the current edge 
+                if (curOrientation == firstOrientation) != (curDir == firstDir):	# i.e. xor
```

Comment:
Again this seems to be indented 8 spaces. The inner tests seem to be correctly indented, but the first branch is indented 8 spaces.